Cargo.toml

```diff
@@ -9,6 +9,7 @@ clap = "2.33.0"
 lazy_static = "1.3.0"
 log = { version = "0.4.8", features = ["std"] }
 vmm = { path = "vmm" }
+libc = "0.2.60"
```

Comment:
Is this still needed now you removed the tests?

Reply:
Not needed anymore.